emacs-with-nyxt.el

```diff
@@ -69,9 +69,7 @@ Defaults to Sly because it has better integration with Nyxt."
   "Send STRING to `cl-ide'."
   (cond
    ((eq cl-ide 'slime) (slime-repl-eval-string string))
-   ((eq cl-ide 'sly) (sly-eval-with-transcript
-                      `(slynk:interactive-eval-region
-                        ,string)))))
+   ((eq cl-ide 'sly) (cdr (sly-eval `(slynk:eval-and-grab-output ,(s-concat "(progn " string ")")))))))
```

Comment:
this is good, thanks!
Just for your elisp culture (maybe you know that already) you can also use `(format "(progn %s)" string)`

Reply:
I always forget about format